site/quickstart.md

```diff
@@ -170,11 +174,19 @@ domain namespace (`sample-domain1-ns`) and the `domainHomeImageBase` (`oracle/we
 For example, assuming you named your copy `my-inputs.yaml`:
 ```
 $ cd kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image
-$ ./create-domain.sh -i my-inputs.yaml -o /some/output/directory -u username -p password -e
+$ ./create-domain.sh -i my-inputs.yaml -o /some/output/directory -u weblogic -p welcome1 -e
```

Comment:
Why did we change the username and password to specific ones?

Reply:
It was a recommendation to me from Tom M to be more specific and emphasize that the user name and password needed to be the same as the one specified in the WebLogic credentials. (It also fits with the following sentence which refers to it: "You need to provide the WebLogic administration user name and password in the -u and -p options respectively, as shown in the example."